components/webui/common/src/socket.ts

```
@@ -0,0 +1,83 @@
+    "collection::find::update": (respArgs: {
+        queryId: QueryId;
+        data: object[];
+    }) => void;
+}
+
+/**
+ * Empty but required by Socket IO.
+ */
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
+interface InterServerEvents {
+}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)